### PR TITLE
[ONNX] Support aten.stack for dynamo_export

### PR DESCRIPTION
**Pull request by BowenBao**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #99202
* __->__ #99191
* #98421

### CODE REVIEW

File: torch/onnx/_internal/fx/function_dispatcher.py [hunk 1]
```diff
@@ -132,6 +132,7 @@ def aten_getitem(self, i):
     "aten::softmax": ops.special.aten_special_softmax,
     "aten::split": ops.core.aten_split,
     "aten::sqrt": ops.core.aten_sqrt,
+    "aten::stack": ops.core.aten_stack,
```
Comment (BowenBao) [inline torch/onnx/_internal/fx/function_dispatcher.py:135]: @justinchuby iirc this op have been in torchlib for a while. Curious what is the updating routine for bringing in new supported ops in torchlib?

Comment (justinchuby) [reply torch/onnx/_internal/fx/function_dispatcher.py:135]: I don’t think we have a routine for now. When we do dispatch it should become a discovery process

Comment (justinchuby) [reply torch/onnx/_internal/fx/function_dispatcher.py:135]: For now they are just added as needed

**Review by justinchuby — APPROVED** [review]

File: torch/onnx/_internal/fx/function_dispatcher.py (hunk 1 above)
Comment (titaiwangms) [reply torch/onnx/_internal/fx/function_dispatcher.py:135]: We could adopt fx exporter into `test_op_consistency`.

**Pull request closed without merging**